### PR TITLE
Issue #399: New README.md

### PR DESCRIPTION
**Pull request by astormnewrelic**
## Proposed Release Notes

- Streamlined README.md

## Links

- https://github.com/newrelic/node-newrelic/issues/399

## Details

Streamlines README.md by converting it to the New Standard New Relic format, and replacing previous documentation with links to appropriate docs.newrelic.com site.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,745 +1,140 @@
...
-Allows you to explicitly filter out long-polling routes or requests you know
-are going to be time-consuming in an uninteresting way, and also allows you
-to gather metrics for requests that would otherwise be ignored. Note that
-to prevent a transaction from being ignored with this function, you **must**
-pass `false` as the parameter – in this case `null` or `undefined` will be
-ignored.
+- [New Relic's official NodeJS Agent Documentation](https://docs.newrelic.com/docs/agents/nodejs-agent)
 
-#### newrelic.noticeError(error, customParameters)
+- [Developer Docs](http://newrelic.github.io/node-newrelic/docs/)
```
Comment (michaelgoin) [inline README.md:106]: Not entirely clear this contains the instrumentation API documentation but maybe could be inferred.

Maybe something like "Developer Docs and Instrumentation API".

File: README.md [hunk 2]
```diff
@@ -1,745 +1,148 @@
...
-  its own connection to New Relic's servers, and will incur its own overhead
-  costs.
-
-### New Relic features available for other platforms not yet in Node.js
-
-* explain plans
-* thread profiling
-* X-ray transactions (depends on thread profiling)
-* capacity planning
+The NodeJS Agent also uses source code from third-party libraries. You can find full details on which libraries are used and the terms under which they are licensed in [the third-party notices document](/node-newrelic/blob/main/THIRD_PARTY_NOTICES.md).
```
Comment (carlonewrelic) [inline README.md:144]: Third-party notices link appears broken in `rich diff` view.  This is the generated link:  `https://github.com/newrelic/node-newrelic/blob/main/node-newrelic/blob/main/THIRD_PARTY_NOTICES.md`
I think the path to the file should be relative to this one so this should work:  `THIRD_PARTY_NOTICES.md`

File: README.md [hunk 3]
```diff
@@ -1,745 +1,148 @@
...
-so it has a place to set the custom attribute.
+- [API Transaction Naming](https://docs.newrelic.com/docs/agents/nodejs-agent/api-guides/nodejs-agent-api#request-names) and [Rules Based Transaction Naming](https://docs.newrelic.com/docs/agents/nodejs-agent/api-guides/nodejs-agent-api#ignoring)
 
-Example of setting multiple custom attribute at once:
+- [Custom Instrumentation/Transactions](https://docs.newrelic.com/docs/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api#creating-transactions)
 
-```js
-newrelic.addCustomAttributes({test: 'value', test2: 'value2'});
-```
+- [The Changelog](/node-newrelic/blob/main/NEWS.md)
```
Comment (carlonewrelic) [inline README.md:124]: This link does not work in `rich diff` view.  I believe it should just be `News.md`

**Comment by astormnewrelic** [comment]
Changed things up to used language based fenced code blocks (`"```js"` style) and changed links to use fully qualified domain names and avoid all the weirdness with `href`s in different context in GItHub repos.

**Pull request merged**